xc7/primitives/slicem/srl/a_srl.pb_type.xml

```diff
@@ -0,0 +1,65 @@
+<pb_type name="ASRL" num_pb="1" xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
Please add a comment on why the a_srl is special, for future reference.

Reply:
It is special because it requires different pack patterns for chaining. I added comment at the beginning of the file.